libcxx/test/std/time/time.clock/time.clock.gps/types.compile.pass.cpp

```diff
@@ -36,11 +36,11 @@
 #include "test_macros.h"
 
 // class gps_clock
-using rep                = std::chrono::gps_clock::rep;
-using period             = std::chrono::gps_clock::period;
-using duration           = std::chrono::gps_clock::duration;
-using time_point         = std::chrono::gps_clock::time_point;
-constexpr bool is_steady = std::chrono::gps_clock::is_steady;
+using rep                                 = std::chrono::gps_clock::rep;
+using period                              = std::chrono::gps_clock::period;
+using duration                            = std::chrono::gps_clock::duration;
+using time_point                          = std::chrono::gps_clock::time_point;
+[[maybe_unused]] constexpr bool is_steady = std::chrono::gps_clock::is_steady;
```

Comment:
As the `is_steady` is tested only once (ditto `tai_clock::is_steady`) and doesn't seem necessary for twice, should we just remove it and write `LIBCPP_STATIC_ASSERT(std::chrono::gps_clock::is_steady == false);` instead?
CC @mordante.

Reply:
Note that `time.clock.utc/types.compile.pass.cpp` is also affected. (I missed gps and tai in my previous #131787.)

Reply:
I agree it's cleaner, but there are indeed more clocks affected. I've created #132546 to look into this.